sensor/Instructions.md

```diff
@@ -51,14 +51,25 @@ make updaterdk
 
 Next, run `make bin/customsensor` to build the executable.
 
-#### 6. Test your module locally
+#### 5. Test your module locally
 
-Follow the instructions to [Add a local module](https://docs.viam.com/registry/configure/#add-a-local-module).
+While testing, we recommend that you add the resources you've built with a remote to your machine because it allows you to recompile your code without having to restart your entire machine:
 
-#### 7. Upload your module to the registry
+1. Edit the [`cmd/module/cmd.go` file](./cmd/module/cmd.go). The code there should configure and run a robot with your modular resource configured.
+2. If you haven't already got a machine, create a new machine on [the Viam app](app.viam.com) and follow the setup instructions on the setup tab.
```

Comment:
```suggestion
2. If you don't already have a machine, create a new machine on [the Viam app](app.viam.com) and follow the setup instructions.
```
Setup is no longer a "tab" with R2D2

Reply:
Secondly, why are you having them create a machine if they're making a remote just for testing. If they're going to actually run viam-server (per the setup instructions) then they can just configure their module, and not mess with a remote at all. The reason a "remote" makes testing easier is you can directly use the local control (web) page without having to configure a real robot or mess with App at all.

Reply:
When I tried this the local control page at localhost:8083 did not show the sensor. It only shows operations and sessions
![Screenshot 2024-03-22 at 17 49 21](https://github.com/npentrel/module-templates/assets/5212232/3d6e85e6-d059-485d-8a67-da47de0e33bc)

Should it show the sensor there?